Content/0.4.13 - Device Factor Sources/deviceFactorSources.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Added them in a new file, rather than updating [Seed Phrases](https://github.com/radixdlt/apps-localization-src/blob/main/Content/0.4.8%20-%20Seed%20Phrases/seedPhrases.json), since most of the factor sources have a seed phrase linked, so better call it the same way we call the `FactorSourceKind`